multiversx_sdk_core/interfaces.py

```diff
@@ -14,6 +14,8 @@ def bech32(self) -> str: ...
 ITransactionOptions = int
 ISignature = bytes
 ITokenIdentifier = str
+IGasPerDataByte = int
+IGasPriceModifier = float
```

Comment:
For this one, we don't need an interface - can simply be a float.

Reply:
Modified it to be of type `float`